examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>initialization takes more time .. (see cells with inline magic <code>%%time</code>)</p><p> </p><p><code>Using multiple GPUs is a convenient way of speeding up th model training process</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>